msteams-platform/bots/how-to/conversations/bot-copilot-handoff.md

```diff
@@ -0,0 +1,178 @@
+   { 
+
+   "type": "Action.OpenUrl", 
+
+   "title": "Handoff to Bot", 
+
+   "url": "https://teams.microsoft.com/l/chat/0/0?users=${botMri}&continuation=${continuationToken}" 
+   }
+   ```
+
+   **Parameters**
+
+   |Property |Description  |
+   |---------|---------|
+   |`botMri`     |The `botMri` must be in the format `28:<botID>`. The botMri specifies the bot chat that the user is redirected to when they select the action button that contains the deep link url.        |
+   |`continuationToken`     | The `continuationToken` value can be any string, but ensure that the total URL length doesn't exceed 2048 characters and the string is URL encoded.        |
+   |`Action.OpenUrl`     |  The `Action.OpenUrl` property allows the user to handoff the conversation to a bot.       |
+
+   **Sample payload**
+
+   ```json
+   { 
+    "name": "handoff/action", 
+    "type": "invoke", 
+    "timestamp": "2024-04-15T19:50:32.945Z", 
+    "localTimestamp": "2024-04-15T19:50:32.945Z", 
+    "id": "f:ad06278a-0dd1-8811-71b1-f65e2bfd4570", 
+    "channelId": "msteams", 
+    "serviceUrl": "https://smba.trafficmanager.net/amer/", 
+    "from": { 
+        "id": "29:1jzORtjcfpYTLQDR9O4TyLz9LDwHskubQN1Ljc-aFO4L8dnZatjFpSw1PCGa-Mm-Jo4uLp67Lvekcjq2hkPoxdA", 
+        "aadObjectId": "57162b23-ae6f-4e95-9774-27f7cb019864" 
+    }, 
+    "conversation": { 
+        "conversationType": "personal", 
+        "tenantId": "72f988bf-86f1-41af-91ab-2d7cd011db47", 
+        "id": "a:13tOiSzRqeub3zaqoTHKpvOkk8Y1zFxk-g8WKdAUM2tjhTBFMt4RSuL8YWi7uwFNBmbxsyzYYktJEyfimYXYiEoplQ34aJs1y8trDb7EIcG09xOjSUieHVzFZ2b8tkagZ" 
+    }, 
+    "recipient": { 
+        "id": "28:68935e91-ff09-4a33-a675-0fe09f015706", 
+        "name": "NorthwindProducts" 
+    }, 
+    "entities": [ 
+        { 
+            "locale": "en-US", 
+            "country": "US", 
+            "platform": "Android", 
+            "timezone": "America/Chicago", 
+            "type": "clientInfo" 
+        } 
+    ], 
+    "channelData": { 
+        "tenant": { 
+            "id": "72f988bf-86f1-41af-91ab-2d7cd011db47" 
+        }, 
+        "source": { 
+            "name": "message" 
+        }, 
+        "legacy": { 
+            "replyToId": "1:1_qLAAGcfze29QAWxzicc7gvR3vuNAlKvth08vavxYYs" 
+        } 
+    }, 
+    "replyToId": "1713210583687", 
+    "value": { 
+        "continuation": "test-continuation-token" 
+    }, 
+    "locale": "en-US", 
+    "localTimezone": "America/Chicago", 
+    "rawTimestamp": "2024-04-15T19:50:32.945Z", 
+    "rawLocalTimestamp": "2024-04-15T14:50:32.945-05:00", 
+    "callerId": "urn:botframework:azure" 
+   }
+   ```
+
+   The `Action.OpenUrl` property allows the user to handoff the conversation to a bot.  After the user is redirected to the bot chat, the bot receives the `continuationToken` value as an invoke message of the type `handoff/action`, if the app associated with the bot is installed in the user's personal scope and the continuationToken value isn't empty.
```

Comment:
"hand off", since it's used as a verb here.

Reply:
updated.